tests/numeric/test_ScalingTransformer.py

```diff
@@ -123,7 +123,7 @@ def test_super_init_called(self, mocker):
 
 
 class TestCheckNumericColumns:
```

Comment:
B: we now set up our test classes to inherit a set of generic tests from other test modules. See the guidance in the issue #275 #268  for an example in this numeric module.  You need to set up the test classes for init, fit and transform to inherit from a parent test class and add 

   `@classmethod
    def setup_class(cls):
        cls.transformer_name = "ScalingTransformer"`
        
 to each so that the inherited tests can function (they look up initialised transformers and minimal dataframes from conftest.py using this)
 
 After doing this check_numeric will be covered by tests in BaseNumericTransformerFitTests and BaseNumericTransformerTransformTests so these tests can be deleted form here once those tests have been inherited.
 
 You can also delete any implementation tests so we are just testing functionality.  There should be far fewer tests in this module when you are done!